packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -106,12 +106,6 @@
                 position: unset;
                 width: 90vw;
             }
-
-            &--ios {
```

Comment:
Please remove the class from the component as well.

Reply:
sure